Lifestyle/todo.30s.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Track todos in a simple ~/.todo file
+#
+
+todo_file="$HOME/.todo"
+count=$(wc -l $todo_file | awk '{print $1}')
+echo "Todos: $count"
+echo "---"
+cat $todo_file
```

Comment:
```
In foo.sh line 10:
cat $todo_file
    ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Before:
```sh
cat $todo_file
```

After:
```sh
cat "$todo_file"
```